src/main/java/org/jenkinsci/plugins/junitrealtimetestreporter/RealtimeJUnitStep.java

```diff
@@ -142,7 +142,11 @@ public Long getParseInterval() {
 
     @DataBoundSetter
     public void setParseInterval(Long parseInterval) {
-        this.parseInterval = parseInterval;
+        if (parseInterval == 0L) {
```

Comment:
Won't this cause a `NullPointerException` if `parseInterval` is `null`?

Reply:
Yes you are right -> https://github.com/jenkinsci/junit-realtime-test-reporter-plugin/pull/96